manager/orchestrator/task.go

```diff
@@ -119,6 +119,10 @@ func nodeMatches(s *api.Service, n *api.Node) bool {
 		return false
 	}
 
+	if s.Spec.Task.Placement != nil {
```

Comment:
Is the logic flipped here? I think this should be;

```suggestion
	if s.Spec.Task.Placement == nil {
```

Reply:
Oops. Thanks